app/models/order.rb

```diff
@@ -7,17 +7,33 @@ class Order < ActiveRecord::Base
   validates :status, presence: true
   validates :subtotal, presence: true
 
-  enum status: %w(ordered completed cancelled)
+  enum status: %w(ordered completed cancelled paid)
 
-  def self.currently_ordered
+  def self.number_currently_ordered
     ordered.count
   end
 
-  def self.currently_completed
+  def self.number_currently_completed
     completed.count
   end
 
-  def self.currently_cancelled
+  def self.number_currently_cancelled
     cancelled.count
   end
+
+  def date_of_order(order_created_at)
+    order_created_at.strftime("%A, %d %b %Y %l:%M %p")
+  end
+
+  def total
+    self.subtotal
+  end
+
+  def show_status
+    if status == "ordered"
+      status
+    else
+      "#{status} -  updated on: #{self.updated_at.strftime("%A, %d %b %Y %l:%M %p")}"
```

Comment:
Line is too long. [85/80]<br>Redundant `self` detected.<br>Prefer single-quoted strings inside interpolations.